### ISSUE TITLE
Please add option to ignore rotation info in metadata

### ISSUE THREAD

**Issue by pentool** [mifi/lossless-cut#1914]
### The fewer issues I have to read, the more new features I will have time to implement, so I ask that you please try these things first

- [X] Check out all the menus and keyboard shortcuts. Maybe the feature already exists?
- [X] Try with the [newest version from GitHub](https://github.com/mifi/lossless-cut/releases/latest)
- [X] Read the [documentation](https://github.com/mifi/lossless-cut) and [FAQ, Known issues, Troubleshooting](https://github.com/mifi/lossless-cut/blob/master/issues.md) to see if the feature already exists
- [X] See if someone already suggested this under [Issues](https://github.com/mifi/lossless-cut/issues) or [Discussions](https://github.com/mifi/lossless-cut/discussions)

### Description

Some of the footage in my action cam was created while the camera was up side down. When you rotate the camera 180 degrees you can select this orientation to be the "correct" orientation so you will see the footage correctly (right side up) both in the camera's display and in any video player.

Such footage has a rotation info in the metadata with the value of -180, eg:

`
side_data_list: [
{
side_data_type: 'Display Matrix',
displaymatrix: '\n00000000:  -65536  0  0\n00000001:  0  -65536  0\n00000002:  125829120  70778880  1073741824\n',
rotation: -180,
},
],
`

So I have two footage.  One is shot normally (Footage A), and the other was shot with the camera inverted (Footage B) but marked in the camera as "correct orientation". When you view the footage which was shot with the camera up side dow in a video player, it plays back the footage correctly (right side up). So it is not inverted. When you view the footage in LosslessCut, it also plays back correctly (right side up). So far so good.

However, when you merge Footage A (shot with the camera right side up) and Footage B (shot with the the camera up side down), the merged footage will play back Footage A part right side up, and Footage B part up side down.

Since all individual footage plays back right side up, it is difficult to tell which footage was actually shot with the camera being up side down, unless you check the metadata of every footage and see if the rotation value of -180 is present.

If there were a setting to ignore rotation metadata when merging mixed footage, that would help a lot and everything would be quick and easy. If you only have about 5 or so videos it's ok, but when there are more then 100, that's gonna be a chore and a headache to fix.

Thank you.

**Comment by mifi** [comment]
Not sure what you want to do here. You want to merge up-side-down footage with non-upside-down footage? that is not possible to do correctly because it's not possible for the rotation metadata to change over the course of a video. Or do you want to take a folder of 100 videos and just remove all the rotation metadata from all the videos to make them show in their original raw recorded orientation? if so, I think it can be easily achieved with a bash command:
```bash
for file in *.mp4; do ffmpeg -i "$file" -c copy -metadata:s:v:0 rotate=0 "$file"-out.mp4; done
```

**Comment by pentool** [comment]
Thank you, I will look into trying this method.

**Comment by pentool** [comment]
I have concluded that sadly, it is not possible to invert a video just by setting the 'rotation' flag to zero and concatenate/merge it with a video that was filmed right side up (aka normal). I have tried setting the 'rotation' flag to zero. I have tried rotating the video in LosslessCut and export it. Regardless how I slice it, when I merge a regular video (where the camera was right side up) with a vide where the camera was up side down, when you merge/concatenate them they will end up just like that. One being right side up and the other is being up side down. The only way out of this is recompressing the up side down video either in an app like HandBrake or concatenate the two videos in an editing software.
